step/src/test/java/com/google/sps/DatabaseHandlerTest.java

```diff
@@ -0,0 +1,53 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps;
+
+import java.util.HashSet;
+import java.util.Arrays;
+import java.util.NoSuchElementException;
+import org.junit.Assert;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+import org.apache.commons.collections4.CollectionUtils;
+
+/** */
+@RunWith(JUnit4.class)
+public final class DatabaseHandlerTest {
+
+  private static final User USER_A = new User(1, "userA@email.com", "User A");
+
+  @Test
+  public void testAddAndGetUser() {
```

Comment:
Can you add comments describing what each test is doing? It'll make it much easier to understand what each test is testing for.